src/Microsoft.Performance.Toolkit.Engine/Engine.cs

```diff
@@ -863,10 +956,38 @@ private RuntimeExecutionResults ProcessCore()
                 this.dataSourcesToProcess);
 
             var executors = CreateExecutors(allDataSourceAssociations);
+
+            var extendedTables = new HashSet<ITableExtensionReference>();
+            var processorTables = new Dictionary<TableDescriptor, ICustomDataProcessor>();
+
+            foreach (var table in this.enabledTables)
+            {
+                foreach (var executor in executors.Where(x => x.Context.CustomDataSource.AvailableTables.Contains(table)))
+                {
+                    executor.Processor.EnableTable(table);
+                    processorTables.Add(table, executor.Processor);
+                }
```

Comment:
I'm wondering if this should be part of an "else" block below the if block below. Given the a table is uniquely identified by a Guid, it would be a bug to be in both sets.